server/dash/dashboards/neighborhood_recent.py

```diff
@@ -176,7 +158,10 @@ def update_table(selected_council):
 def update_text(selected_council):
     create_count = df.query(f"councilName == '{selected_council}' and createdDate >= '{start_date}'")['srnumber'].count()  # noqa
     close_count = df.query(f"councilName == '{selected_council}' and closedDate >= '{start_date}'")['srnumber'].count()  # noqa
-    return create_count, close_count, create_count - close_count
+    if create_count == 0 and close_count == 0:
```

Comment:
Is this check necessary? It seems like the original return statement ends up being the same when create_count and close_count are 0.

Reply:
My bad - I was suppose to check the condition when there is close count > 0 even when create count == 0. There is a data quality issue where a  request has closing date in the future (year field is probably mistyped). I was trying to accomodate that.